app/assets/stylesheets/admin/loans/_charts.scss

```diff
@@ -1,3 +1,23 @@
-.chart {
-  text-align: center;
+.breakeven-chart {
+  margin: 2em .5em 1em;
+  page-break-before: always;
+
+  .section-title, .chart-title {
+    display: block !important;
```

Comment:
I prefer not to use importants, but the tree that the questions are loaded into has some strange styling. I can add more specificity to these elements, if preferred.